package.json

```diff
@@ -5,11 +5,12 @@
   "main": "index.js",
   "scripts": {
     "coverage": "nyc report --reporter=text-lcov | coveralls",
-    "test": "set NODE_ENV=test&& nyc mocha --require @babel/register 'src/**/**.test.js' --no-timeout ",
-    "dev-start": "nodemon --exec babel-node src/index.js",
+    "pretest": "NODE_ENV=test yarn drop && yarn migrate",
```

Comment:
there is no need for the `pretest` script. add the content of this script to the `test` script

Reply:
I've resolved it. Thanks.